Automattic/WP-Job-Manager
# Update Twitter Logo to X

Fixes #2589 

### Changes Proposed in this Pull Request

* Replace old bird logo with the new X logo used by what once was called Twitter
* Re-add SVG version of the font, for the purpose of allowing easy font editing
* Update `config.json` file saved by Fontello with the new custom icons for WPJM and Twitter/X

### Testing Instructions

1. Run `npm start` on the project to update the assets/fonts
2. Verify if the icons on WPJM still appear correctly
3. Verify if, when defining Company Name + Company Twitter on a job listing, the new x icon will appear correctly on the frontend


### Release Notes

* Replace old bird logo used by Twitter with the new X logo

### Screenshot / Video

![Screenshot with the new Twitter icon](https://github.com/Automattic/WP-Job-Manager/assets/529864/3e434b84-2c8d-480c-a59e-2c4023ecef4f)

### Notes

I'd like feedback on the release note written above. Is it a good release note? I'm not sure.


----

| Plugin build for 9da982c59312e6829ff9d3badbedf02a6239ca17 <a href="#"><img width=600></a> |
| ------------------------------------------------------------ |
| 📦 [Download plugin zip](https://wpjobmanager.com/wp-content/uploads/2023/11/wp-job-manager-zip-2644-9da982c5.zip)                       |
| ▶️ [Open in playground](https://wpjobmanager.com/playground/?core=2023/11/2644-9da982c5)             |
